### PR TITLE
Nt/tweaks

### PR DESCRIPTION
**Pull request by noumantahir**
### What does this PR?

- pin code modal fixes to achieve expected behaviour - check video
- themefied input and welcome modal

### Screenshots/Video
https://user-images.githubusercontent.com/6298342/111686188-9729f980-884a-11eb-9294-6f0e1abd22d7.mp4

### CODE REVIEW

**Comment by noumantahir** [comment]
- handling mismatched pin while creating new pin
- added translation entries

**Comment by noumantahir** [comment]
- fixed issue with login popup appearing upon login after reset,

**Comment by noumantahir** [comment]
- showing pincode popup for re-logging in user that had pin set previously without reset.

**Comment by noumantahir** [comment]
- patched up issue logging out on login using pin code

**Summary**
Issue was from loginContainer, the pinCode screen was being called only for verification while essentially user logging in with pinCode means the keys should also update according user set pin code.

Now I am also setting new keys in realm on login with pin code.

The reason I am calling this a patch is because this logout issue was taking too much time, when I figured the issue I just tried patch as cleanly as possible. but overall, the whole pinCodeContainer class can be cleaned to become something very simple.

**Comment by noumantahir** [comment]
updated crypto.js to avoid UTF-8 malfunction issue during pin code verification

**Comment by noumantahir** [comment]
Improved encryption and handling failure for pin code related decryptions.

File: src/config/locales/en-US.json [hunk 1]
```diff
@@ -360,6 +360,7 @@
     "remove_alert": "Are you sure you want to remove?",
     "clear_alert": "Are you sure you want to clear?",
     "clear_user_alert": "Are you sure you want to clear all user data?",
+    "decrypt_fail_alert":"Bad previous app data, you will have relogin to reset app data",
```
Comment (feruzm) [inline src/config/locales/en-US.json:363]: let's change string to this: `Corrupt app state, please relogin to reset state.`

**Comment by noumantahir** [comment]
done

**Comment by noumantahir** [comment]
- better handling decryption exception
- stamping new encryptions

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**